docs/src/index.jsx

```diff
@@ -28,6 +28,8 @@ const onUpdate = () => {
     }
 };
 
+window.theme = theme;
```

Comment:
This is for reference. i.e Aaron can come in and check the values easily.

Reply:
Let's maybe add a comment there to explain why.
Also, should we "namespace" it at all?  e.g. `window.saddleback.theme = theme;`  My understanding is it's considered somewhat bad form to "pollute" the global namespace.  Obviously, we're not using a whole bunch of other frameworks or libraries or anything that should be adding stuff to there that would conflict, but ... you know ... just in case. :-)
NAB

Reply:
Fresh ping on this! :-)
Any objections to placing a quick, short and sweet explanatory comment here?  Also, thoughts about the namespace issue?

It remains a NAB, but perhaps a nice to have.

Reply:
Addressed in another PR that was merged into `dev`. Thank you @groberts314!

Reply:
Looks like that is happening now up on Lines 14-19.  14-18 are the shiny new explanatory comment and 19 is this same assignment.

Can we remove Line 38 then?

Reply:
Fixing...

Reply:
Removed this one in 207cc05.  Likely the duplicate ones were due to a previous merge.  In any event, it's good now, and I am resolving this thread.